Guessing Game/index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <title>Guessing Game</title>
+    <script src="./GuessingGame.js"></script>
+  </head>
+  <body>
+    <div>
+      <h1>Guess the Number</h1>
+      <button onclick="getUserNumber()">Calculate</button>
```

Comment:
Calculate is not what a "Guess the number" button should be associated with the game title.